spec/requests/registrations_spec.rb

```diff
@@ -0,0 +1,18 @@
+RSpec.describe 'Registrations', type: :request do
+  describe 'POST /registrations' do
+    it 'works! (now write some real specs)' do
+      get '/'
+      expect(response).to render_template('index')
```

Comment:
I don't see a need for this assertion.

Reply:
It was to confirm a request beforehand...but I can keep things to single spec per request.